src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md

```diff
@@ -0,0 +1,148 @@
+for Chrome is one tool in the [Web Vitals toolbox](https://web.dev/vitals-tools/) that focuses on
+giving you performance data about the pages you visit as you browse the web. But you may be
+wondering, "how typical are user experiences like mine?" The last thing we want to do is perpetuate
+the myth that _if it's fast for me, it's fast for thee_. So we're launching version 1.0 of the
+Web Vitals extension with a new feature to integrate real-user data from the
+[Chrome UX Report](https://web.dev/chrome-ux-report/) (CrUX) with your local Core Web Vitals
+experiences, putting them in the greater context of how other users have experienced the same pages
+and websites. It comes with a sparkly new UI and I'm excited to show you how it works.
+
+{% Video
+  src="video/STd8eW8CSiNp5B1bX0R6Dww2eH32/EFeVK6dAwBZWzKhqBd1Y.mp4",
+  height="493",
+  width="800",
+  controls="true",
+  loop="true",
+  muted="true",
+  preload="auto",
+  class="w-screenshot"
+%}
+
+CrUX is a public dataset of real-user experiences in Chrome. It powers some of the critical tools
+in the Core Web Vitals workflow like [Search Console](https://support.google.com/webmasters/answer/9205520#about_data)
+and [PageSpeed Insights](https://developers.google.com/speed/docs/insights/v5/about#crux).
+The raw data for millions of websites is also publicly queryable in the [CrUX dataset on BigQuery](https://web.dev/chrome-ux-report-bigquery)
+and the [CrUX API](https://web.dev/chrome-ux-report-api). For this update to the Web Vitals
+extension, we've integrated it with the page and origin-level desktop data in the CrUX API. User
+experience data is broken down into three qualitative ratings: good, needs improvement, and poor.
+The thresholds used for each rating are documented in the guides for each of the Core Web Vitals
+metrics: [LCP](https://web.dev/lcp/#what-is-lcp), [FID](https://web.dev/fid/#what-is-fid), and
+[CLS](https://web.dev/cls/#what-is-a-good-cls-score). So for example, the CrUX API can tell us that
+16% of real-user experiences on a given page are evaluated as having good
+<span style="white-space: nowrap;">LCP ≤ 2.5</span> seconds.
+
+<figure class="w-figure">
+{% Img src="image/STd8eW8CSiNp5B1bX0R6Dww2eH32/EaVNEuQ6gxVLtHYNkSZj.png", alt="Screenshot of the
+Web Vitals extension showing an explanation of how the local LCP experience relates to real-user
+desktop data from the field.", width="800", height="549" %}
+  <figcaption class="w-figcaption">
+    Screenshot of the Web Vitals extension showing an explanation of how the local LCP experience
+    relates to real-user desktop data from the field.
+  </figcaption>
+</figure>
+
+The way this is integrated with the Web Vitals extension is by layering your local experience on
+top of the broader distribution of real-user experiences. It's like when you're in an expansive
```

Comment:
```suggestion
In the new Web Vitals extension interface, your local experience is layered on
top of the broader distribution of real-user experiences. It's like when you're in an expansive
```

Reply:
Again, I'd replace "experience" with "data" or "measurements" here if that works. Not sure about technical correctness, so I'll leave it to you to decide.

Reply:
Ah, looking closely at the extension UI, I see it uses this wording. I won't edit it anymore, though I still suggest keeping the edit in the intro. Intro is the hook, better to be simple even if it's not quite technically correct.